CMakeLists.txt

```diff
@@ -41,20 +47,43 @@ set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER "CMake Targets")
 project(leptonica C CXX)
 
 set(VERSION_MAJOR 1)
-set(VERSION_MINOR 73)
-set(VERSION_PLAIN ${VERSION_MAJOR}.${VERSION_MINOR})
-
-find_package(GIF)
-find_package(JPEG)
-find_package(PNG)
-find_package(TIFF)
-find_package(ZLIB)
+set(VERSION_MINOR 74)
+set(VERSION_PATCH 2)
+set(VERSION_PLAIN ${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH})
+
+option(BUILD_PROG "Build utility programs" OFF)
+
+if(NOT EXISTS ${PROJECT_SOURCE_DIR}/.cppan)
+    find_package(GIF)
+    hunter_add_package(Jpeg)
+    find_package(JPEG)
```

Comment:
Please check wiki for usage instructions. E.g. it should be `find_package(JPEG CONFIG REQUIRED)` for Jpeg:
* https://github.com/ruslo/hunter/wiki/pkg.jpeg